README.md

```diff
@@ -8,17 +8,55 @@ This operator uses the [Go Operator Framework](https://sdk.operatorframework.io/
 
 The operator is NOT functional at this stage.
 
+By default, the operator is deployed in namespace "network-observability".
+
+## Deploy an existing image
+
+Images are built and pushed through CI to [quay.io](https://quay.io/repository/netobserv/network-observability-operator?tab=tags). To refer to the latest version of the `main` branch, use `IMG=quay.io/netobserv/network-observability-operator:main` or simply `VERSION=main`. To refer to older versions, use the commit short-SHA as the image tag.
+
+E.g. to deploy the latest build of branch `main`:
+
+```bash
+VERSION=main make deploy
+```
+
 ## Build / push / deploy
 
+The repository `quay.io/netobserv/network-observability-operator` is only writable by the CI, so you need to use another repository (such as your own one) if you want to use your own build.
+
+For instance, to build from a pull-request, checkout that PR (e.g. using github CLI or `git fetch upstream pull/99/head:pr-99 && git checkout pr-99` (replace `99` with the PR ID)), then run:
+
 ```bash
-# With default image to quay.io/netobserv
-make image-build image-push deploy
+IMG="quay.io/youraccount/network-observability-operator:v0.0.1" make image-build image-push deploy
+```
+
+Note, the default image pull policy is `IfNotPresent`, so if you previously deployed the operator on a cluster and then create another build with the same image name/tag, it won't be pulled in the cluster registry. So you need either to provide a different image name/tag for every build, or modify [manager.yaml](./config/manager/manager.yaml) to set `imagePullPolicy: Always`, then re-deploy.
+
+Then, you can deploy a custom resource, e.g.:
 
-# With custom image
-IMG="docker.io/myuser/netobserv:latest" make image-build image-push deploy
+```bash
+kubectl apply -f ./config/samples/flows_v1alpha1_flowcollector.yaml
```

Comment:
need to apply namespace here "-n network-observability"

Reply:
actually, here there's no need to provide a namespace because this resource is cluster-scope, not namespace-scope